.eslintrc.js

```diff
@@ -3,12 +3,14 @@ require("@jobber/eslint-config/patch-eslint-plugin-resolution.js");
 
 const packageAliases = [
   ["@jobber/components", "./packages/components/src"],
+  ["@jobber/components-native", "./packages/components-native/src"],
```

Comment:
are we going for 

```js
import { Icon } from "@jobber/components-native"
```

if so, we don't need this.

(PS: I also added this, I'm sorry 😬 )